adagrams/game.py

```diff
@@ -1,13 +1,120 @@
 from random import randint
 
+LETTER_POOL = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B",
+            "C", "C", "D", "D", "D", "D",
+            "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E",
+            "F", "F", "G", "G", "G", "H", "H",
+            "I", "I", "I", "I", "I", "I", "I", "I", "I",
+            "J", "K", "L", "L", "L", "L",
+            "M", "M", "N", "N", "N", "N", "N", "N",
+            "O", "O", "O", "O", "O", "O", "O", "O",
+            "P", "P", "Q",
+            "R", "R", "R", "R", "R", "R",
+            "S", "S", "S", "S",
+            "T", "T", "T", "T", "T", "T",
+            "U", "U", "U", "U", "V", "V",
+            "W", "W", "X", "Y", "Y", "Z"]
+
+SCORE_CHART = {
+        "A" : 1,
+        "E" : 1,
+        "I" : 1,
+        "O" : 1,
+        "U" : 1,
+        "L" : 1,
+        "N" : 1,
+        "R" : 1,
+        "S" : 1,
+        "T" : 1,
+        "D" : 2,
+        "G" : 2,
+        "B" : 3,
+        "C" : 3,
+        "M" : 3,
+        "P" : 3,
+        "F" : 4,
+        "H" : 4,
+        "V" : 4,
+        "W" : 4,
+        "Y" : 4,
+        "K" : 5,
+        "J" : 8,
+        "X" : 8,
+        "Q" :10,
+        "Z" :10
+    }
```

Comment:
I would condense this to a few lines that way our codebase could look a more uniform since we did `LETTER_POOL` like this. 
```suggestion
SCORE_CHART =  {
    "A": 1, "E": 1, "I": 1, "O": 1, "U": 1,
    "L": 1, "N": 1, "R": 1, "S": 1, "T": 1,
    "D": 2, "G": 2,
    "B": 3, "C": 3, "M": 3, "P": 3,
    "F": 4, "H": 4, "V": 4, "W": 4, "Y": 4,
    "K": 5,
    "J": 8, "X": 8,
    "Q": 10, "Z": 10
}
```